src/scrapers/base-isracard-amex.ts

```diff
@@ -61,6 +61,11 @@ interface ScrapedAccount {
   processedDate: string;
 }
 
+interface AccountCredit {
+  creditUtilization: number;
+  creditLimit: number;
+}
```

Comment:
I think you can declare this interface once in `transactions.ts` and import it everywhere.

Reply:
I though about that too, but as this is an internal type (it's used locally, not "passed" outside the file) I opted into this.
If you think otherwise - let me know.